src/state/inputs.ts

```diff
@@ -15,7 +15,14 @@ import { CLEARED_SIZE_INPUTS, CLEARED_TRADE_INPUTS } from '@/constants/trade';
 
 import { safeAssign } from '@/lib/objectHelpers';
 
-type TradeFormInputs = typeof CLEARED_TRADE_INPUTS & typeof CLEARED_SIZE_INPUTS;
+type TradeFormInputs = {
+  limitPriceInput: string | number;
```

Comment:
I'm not convinced this is safe since this.stateManager.trade in setTradeValue in AbacusStateManager says it takes string values only.

Reply:
I guess this technically works because in abacus despite typing it as string it eventually calls `Parser.asString` which doesn't strictly require it be a string and will just do a little `return "$data"` which is some kind of valid stringifying I guess.

Reply:
But seems unsafe to assume it'll keep doing this.